tests/testthat/test-timeseriessummary.R

```diff
@@ -1587,5 +1581,555 @@ test_that('constructTSDetails properly constructs the two tables for the TSS det
+                                 "preProcessing": [],
+                                 "corrUrl": {
+                                 "urlReportType": "correctionsataglance",
+                                 "url": "https://cida-eros-aqcudev.er.usgs.gov:8444/aqcu-webservice/service/reports/correctionsataglance/?endDate=2017-02-28Z&station=01069500&startDate=2017-01-01Z&primaryTimeseriesIdentifier=efb88ed6b5dc41dcaa33931cd6c144a2"
+                                 }
+                                 }
+}')
+  corrections <- repgen:::parseTSSProcessingCorrections(fillGapsJson, "normal", timezone)
+  expect_equal(corrections[['formattedParameters']], "Fill Gaps Resample Period PT30M; Gap Limit MaxDuration")
+  expect_equal(names(corrections),c("appliedTimeUtc","comment","startTime","endTime","type","user","processingOrder","ResamplePeriod","GapLimit","timezone","formattedParameters"))
+})
+
+test_that('formatCorrectionsParamDeviation correctly formats deviation parameters data ', {
+  timezone <- "Etc/GMT+5"
+  deviationJson <- fromJSON('{
+                           "corrections": {
+                           "normal": [
+                            {
+          "appliedTimeUtc": "2017-06-14T13:31:17.479716Z",
+                            "comment": "Outlier trim",
+                            "startTime": "2017-02-10T08:00:00-05:00",
+                            "endTime": "2017-02-10T23:30:00.0000001-05:00",
+                            "type": "Deviation",
+                            "parameters": {
+                            "DeviationValue": 0.01,
+                            "DeviationType": "DeviationFromMinimum",
+                            "WindowSizeInMinutes": 15
+                            },
+                            "user": "lflight",
+                            "processingOrder": "NORMAL"
+                            }
+                           ],
+                           "postProcessing": [],
+                           "preProcessing": [],
+                           "corrUrl": {
+                           "urlReportType": "correctionsataglance",
+                           "url": "https://cida-eros-aqcudev.er.usgs.gov:8444/aqcu-webservice/service/reports/correctionsataglance/?endDate=2017-02-28Z&station=01069500&startDate=2017-01-01Z&primaryTimeseriesIdentifier=efb88ed6b5dc41dcaa33931cd6c144a2"
+                           }
+                           }
+}')
+  corrections <- repgen:::parseTSSProcessingCorrections(deviationJson, "normal", timezone)
+  expect_equal(corrections[['formattedParameters']], "Deviation type DeviationFromMinimum; value: 0.01, window size 15 minutes")
+  expect_equal(names(corrections),c("appliedTimeUtc","comment","startTime","endTime","type","user","processingOrder","DeviationValue","DeviationType","WindowSizeInMinutes","timezone","formattedParameters"))
+})
+
+test_that('unNestCorrectionParameters correctly formats empty parameters data ', {
```

Comment:
should we also add a test for the unknown parameter type? The switch statement for `formattedParameters` just returns whatever appears in `type`